src/client/driver/index.js

```diff
@@ -173,6 +174,16 @@ export default class ClientDriver {
             });
     }
 
+    _onResizeWindowCommand (command) {
```

Comment:
OK, got it now. But why we need to pass `resize` info through the client? All we need to do is just change title and then change it back. I guess we will use same machinery for screenshots? Can we just save `pendingBrowserWindowCommand` in TestRun (which is screenshot or resize window), then run some kind of `ExecutePendingBrowserWindowCommand` which will change title and trigger TestRun which will run appropriate code and then change title back.